Woof/WoofTests/ViewModel/DetailPetSitterViewModel/DetailPetSitterViewModel+CopyToClipboardTextTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+// swiftlint:disable:next type_name
+final class DetailPetSitterViewModelCopyToClipboardTextTests: XCTestCase {
+    func testThePassedTextIsCopiedToClipboard() {
+        // Given
+        let textToCopy = "a"
+
+        // When
+        testViewModel.copyToClipboardText(textToCopy)
+
+        // Then
+        XCTAssertEqual(UIPasteboard.general.string, textToCopy)
+    }
```

Comment:
Does it make sense to create private computed property
``` swift   
private var clipboardValue: String? {
        UIPasteboard.general.string
    }
```
and use it in tests like this
```suggestion
        // Then
        XCTAssertEqual(clipboardValue, textToCopy)
    }
```

Reply:
Fixed